File: Lab2Scripts/LAB2JAVA/Fecha.java
public class Fecha {
    private int dd;
    private int mm;
    private int aa;

    // Constructor por defecto
    public Fecha() {
        this.dd = 1;
        this.mm = 1;
        this.aa = 2000;
    }

    // Constructor con parámetros
    public Fecha(int dd, int mm, int aa) {
        this.dd = dd;
        this.mm = mm;
        this.aa = aa;
    }

    // Setters
    public void setDia(int dd) {
        this.dd = dd;
    }

    public void setMes(int mm) {
        this.mm = mm;
    }

    public void setA(int aa) {
        this.aa = aa;
    }

    // Getters
    public int getDia() {
        return dd;
    }

    public int getMes() {
        return mm;
    }

    public int getA() {
        return aa;
    }

    // Método toString
    public String toString() {
        return String.format("%02d - %02d - %04d", dd, mm, aa);
    }
}

File: Lab2Scripts/LAB2PYTHON/fecha.py
class Fecha:
    def __init__(self, dd=1, mm=1, aa=2000):
        self._dd = dd
        self._mm = mm
        self._aa = aa

    def set_dia(self, dd):
        self._dd = dd

    def set_mes(self, mm):
        self._mm = mm

    def set_a(self, aa):
        self._aa = aa

    def get_dia(self):
        return self._dd

    def get_mes(self):
        return self._mm

    def get_a(self):
        return self._aa

    def __str__(self):
        return f"{self._dd:02d} - {self._mm:02d} - {self._aa}"
